packages/@o3r/new-version/package.json

```diff
@@ -45,7 +45,9 @@
     "@nx/jest": "~19.5.0",
     "@o3r/build-helpers": "workspace:^",
     "@o3r/eslint-plugin": "workspace:^",
+    "@o3r/schematics": "workspace:^",
     "@o3r/test-helpers": "workspace:^",
+    "@schematics/angular": "~18.2.0",
```

Comment:
Why should we add this one ?

Reply:
It's a mandatory peer dependency of `@o3r/schematics`